packages/babel-plugin-transform-typescript/src/enum.ts

```diff
@@ -118,6 +119,26 @@ function enumFill(path: NodePath<t.TSEnumDeclaration>, t: t, id: t.Identifier) {
   };
 }
 
+export function isSyntacticallyString(expr: t.Expression): boolean {
+  // @ts-ignore(Babel 7 vs Babel 8) Type 'Expression | Super' is not assignable to type 'Expression' in Babel 8
+  expr = skipTransparentExprWrapperNodes(expr);
+  switch (expr.type) {
+    case "BinaryExpression": {
+      const left = expr.left;
+      const right = expr.right;
+      return (
+        expr.operator === "+" &&
+        (isSyntacticallyString(left as t.Expression) ||
+          isSyntacticallyString(right))
+      );
```

Comment:
We have a test case that covers this branch, right?

Reply:
Yes, we have a test that has a string on the left.